lopezMidterm/src/components/Products.jsx

```diff
@@ -1,13 +1,13 @@
-import React, { useEffect, useState } from 'react';
-import { BrowserRouter as Router, Routes, Route, Link} from 'react-router-dom';
-import useFetch from '../hooks/useFetch';
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import useFetch from './hooks/useFetch.js';
 
-function ProductsList() {
+function Products() {
   const [products, setProducts] = useState([]);
   const { loading, error, fetchData } = useFetch();
 
   useEffect(() => {
-    fetchData('db.json').then((data) => {
+    fetchData('http://localhost:3000/products').then((data) => {
```

Comment:
`fetchData` is not a valid property. Take a look at `useFetch`, there isn't anything called `fetchData`

Take a look at the last example in https://neit.instructure.com/courses/56465/pages/live-coding-creating-a-custom-hook?module_item_id=2530828, I think that will get you back on track.

Please get this fixed ASAP.